multiversx_sdk_cli/cli_shared.py

```diff
@@ -497,6 +496,26 @@ def prepare_chain_id_in_args(args: Any):
         args.chain = proxy.get_network_config().chain_id
 
 
+def get_chain_id(chain_id: str, proxy_url: str) -> str:
+    if chain_id and proxy_url:
+        network_provider_config = config.get_config_for_network_providers()
+        proxy = ProxyNetworkProvider(url=proxy_url, config=network_provider_config)
+        fetched_chain_id = proxy.get_network_config().chain_id
+
+        if chain_id != fetched_chain_id:
+            show_warning(
+                f"The chain ID you have provided does not match the chain ID you got from the proxy. Will use the proxy's value: '{fetched_chain_id}'"
+            )
+        return fetched_chain_id
+
+    if chain_id:
+        return chain_id
+    else:
```

Comment:
`else` statement can be dropped, and remaining lines unindent-ed.

Reply:
indeed, done